test-app/.eslintrc.js

```diff
@@ -24,6 +24,42 @@ module.exports = {
   },
   rules: {},
   overrides: [
+    {
+      files: ['**/*.{js,ts}'],
+      plugins: ['ember'],
+      parser: '@typescript-eslint/parser',
```

Comment:
Why do you add '`typescript-eslint` if there is no TS?

Reply:
Done